src/components/shareville-logo/index.jsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function SharevilleLogo({ onlyIcon, height }) {
+  const width = onlyIcon ? height : height * 4.89;
+
+  if (onlyIcon) {
+    return (
+      <svg viewBox="0 0 166.54 203.5" width={width} height={height}>
+        <defs>
+          <style
+            dangerouslySetInnerHTML={{
+              __html: '.svLogoGradientClass-1{fill:url(#linear-gradient);}.svLogoGradientClass-2{fill:url(#linear-gradient-2);}.svLogoGradientClass-3{fill:url(#linear-gradient-3);}',
```

Comment:
Why `svLogoGradientClass` as name? Is it autogenerated? Would be better if it contain shareville like the class below

Reply:
@codler Fixed this now in a new release. (It's generated and never manually used by devs so not a big deal)

Reply:
no but next time someone generate svg it might get same name and will then conflict.

Reply:
@codler ah I see what you mean. I already customised it by prefixing it with `svLogo`. (by default it was `gradientclass-1`, which is very generic). But good cal!